test/helpers/utils.js

```diff
@@ -77,6 +77,14 @@ const getEpoch = async () => {
   return blockNumber.div(EPOCH_LENGTH).toNumber();
 };
 
+const getVote = async (medians) => {
```

Comment:
this is to have random votes ?
this is awesome addition

Reply:
Yes
Helps in accurate representation of how the network will behave with real votes